preop.cpp

```diff
@@ -36,16 +36,15 @@ char *handle_preop (char *ptr) {
 
 	//first get the name
 	name_end = ptr;
-	while (isalpha (*name_end))
+	while (!isspace(*name_end))

```

Comment:
Just talked with @jacobly0 - if `isspace()` hits a NUL character, it's technically not a space, so it will continue to run past the buffer until it hits a NUL (or crashes). Adopting something similar to https://github.com/alberthdev/spasm-ng/blob/master/parser.cpp#L161 for both `isspace()` changes would fix this issue!